eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/VerifyTest.java

```diff
@@ -68,70 +68,70 @@ public void assertThrowsWithCause()
         Verify.assertThrowsWithCause(RuntimeException.class, NullPointerException.class, (Callable<Void>) () -> {
             throw new RuntimeException(new NullPointerException());
         });
-        Verify.assertError(AssertionError.class, () -> Verify.assertThrowsWithCause(RuntimeException.class, NullPointerException.class, (Callable<Void>) () -> null));
```

Comment:
I didn't remember this so I looked it up. AssertionFailedError extends AssertionError, so on the surface, it looks like this assertion might pass. But `Verify.assertError()` delegates to assertSame between the error class files, so it's a very strict assertion. This change makes sense to me.